includes/class-payment-information.php

```diff
@@ -222,4 +231,22 @@ public static function get_token_from_request( array $request ) {
 
 		return $token;
 	}
+
+	/**
+	 * Cchanges type type of the payment.
+	 *
+	 * @param string $type The new type.
+	 */
+	public function set_payment_type( $type ) {
+		$this->payment_type = $type;
+	}
```

Comment:
That piece confuses me. We expect the `Payment_Information->payment_type` to be of type `WCPay\Payment_Type` but use string here. Can we use `Payment_Type` everywhere for type safety?

Reply:
Fixed in https://github.com/Automattic/woocommerce-payments/pull/979/commits/87f84c5f4f2e6403ed2e59429525ab6c85859060